pipelines/config/settings.yml

```diff
@@ -0,0 +1,7 @@
+variables:
+  Unity2018Version: Unity2018.4.6f1

```

Comment:
There are slightly different Unity versions on the internal build machines as the externally-facing ones. I'm not sure the internal machines have this build, and were previously using `.12` (while external used this build)

Reply:
Ah, that's an interesting point. I blamed any variation on copy-paste mistakes as there is a lot of duplication. I would expect that the version should not differ outside of the "internal/external agent pool" split, so we could perhaps have twi sets of those variables (e.g. Unity2018Version and Unity2018VersionAnalog, etc.). I'll see if I can update the config to always pass the version down in template parameters.